nil/tests/rpc_node/call_test.go

```diff
@@ -0,0 +1,143 @@
+package tests
+
+import (
+	"context"
+	"github.com/NilFoundation/nil/nil/common"
+	"github.com/NilFoundation/nil/nil/common/check"
+	"github.com/NilFoundation/nil/nil/common/hexutil"
+	"github.com/NilFoundation/nil/nil/internal/contracts"
+	"github.com/NilFoundation/nil/nil/internal/db"
+	"github.com/NilFoundation/nil/nil/internal/network"
+	"github.com/NilFoundation/nil/nil/internal/types"
+	"github.com/NilFoundation/nil/nil/services/nilservice"
+	"github.com/NilFoundation/nil/nil/services/rpc/jsonrpc"
+	"github.com/NilFoundation/nil/nil/tests"
+	"github.com/ethereum/go-ethereum/crypto"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"github.com/stretchr/testify/suite"
+	"sync/atomic"
+	"testing"
+)
+
+type SuiteRpcNodeCall struct {
+	tests.ShardedSuite
+}
+
+type NetworkManagerFactory = func(ctx context.Context, cfg *nilservice.Config, db db.DB) (network.Manager, error)
```

Comment:
You can define this type in a previous patch right near config - https://github.com/NilFoundation/nil/pull/793/commits/14b46f14b1f5f0e9b072be923e3ba1e915b5987a#diff-3ce05a3487a74b5c8772e50abed56c59cc17d0e7b35b4a8925446661af9fe3a1R90

Reply:
I can't because of cyclic type dependency (`Config` <-> `NetworkManagerFactory`). I mean I can, but I can't refer to it inside `Config`.